ZelBack/src/services/appsService.js

```diff
@@ -9219,10 +9548,26 @@ async function checkMyAppsAvailability() {
         });
       }
     }
+    const minPort = currentHeight.generalScannedHeight >= config.fluxapps.portBlockheightChange ? config.fluxapps.portMinNew : config.fluxapps.portMin - 1000;
+    const maxPort = currentHeight.generalScannedHeight >= config.fluxapps.portBlockheightChange ? config.fluxapps.portMaxNew : config.fluxapps.portMax;
     // choose random port
-    const min = config.fluxapps.portMin - 1000;
-    const max = config.fluxapps.portMax;
+    const min = minPort;
+    const max = maxPort;
+    if (failedPort && previouslyFailedPort === failedPort) {
```

Comment:
This needs to be removed. It was correct like it was before. This way, I can block a range of ports, the test fails, and before it's DOS, the probability of hiting a diferent port is huge and the node will never get DOS.
If there is a port that fails the test, the tests needs to continue to be on same port until it passes.

Reply:
user defined blocked port list added
strict dos implemented